src/views/settings/companies/users/form.vue

```diff
@@ -178,6 +195,17 @@ export default {
     created() {
         this.$store.dispatch("Application/getSettingsLists");
         this.setUser();
+
+        if (this.$route.name === "settingsCompaniesUsersFormEdit") {
+            this.getCompanyUsers().then(({ data: companyUsers }) => {
+                const foundUser = companyUsers.find((user) => user.id === this.$route.params.id);
+                if (isEmpty(foundUser)) {
+                    this.$router.push({ name: "404" });
```

Comment:
We don't want to push the 404 page to the history. 
```suggestion
                    this.$router.replace({ name: "404" });
```

Reply:
Oops...